lib/svg_optimizer/plugins/remove_editor_namespace.rb

```diff
@@ -17,8 +17,27 @@ class RemoveEditorNamespace < Base
         http://www.bohemiancoding.com/sketch/ns
       ]
 
+      def namespaces_to_be_removed
+        xml.namespaces.map do |name, value|
+          _, name = name.split(":")
+          name if NAMESPACES.include?(value)
+        end.compact
+      end
+
+      def remove_namespaced_attributes
+        namespaces_to_be_removed.each do |ns|
+          xml.xpath("//*[@#{ns}:*]").each do |node|
```

Comment:
makes sense to merge all `namespaces_to_be_removed.each` into a single XPath query? This way we would avoid the nested `each` iterator and look through the XML just once.